playground/vanilla/src/main.ts

```diff
@@ -400,6 +400,268 @@ import StoryblokClient from 'storyblok-js-client';
+              },
+              content: [
+                {
+                  type: 'paragraph',
+                  content: [
+                    {
+                      text: 'R2C1',
+                      type: 'text',
+                    },
+                  ],
+                },
+              ],
+            },
+            {
+              type: 'tableCell',
+              attrs: {
+                colspan: 1,
+                rowspan: 1,
+                colwidth: null,
+              },
+              content: [
+                {
+                  type: 'paragraph',
+                  content: [
+                    {
+                      text: 'R2C2',
+                      type: 'text',
+                    },
+                  ],
+                },
+              ],
+            },
+            {
+              type: 'tableCell',
+              attrs: {
+                colspan: 1,
+                rowspan: 1,
+                colwidth: null,
+              },
+              content: [
+                {
+                  type: 'paragraph',
+                  content: [
+                    {
+                      text: 'R2C3',
+                      type: 'text',
+                    },
+                  ],
+                },
+              ],
+            },
+          ],
+        },
+        {
+          type: 'tableRow',
+          content: [
+            {
+              type: 'tableCell',
+              attrs: {
+                colspan: 1,
+                rowspan: 1,
+                colwidth: null,
+              },
+              content: [
+                {
+                  type: 'paragraph',
+                  content: [
+                    {
+                      text: 'R3C1',
+                      type: 'text',
+                    },
+                  ],
+                },
+              ],
+            },
+            {
+              type: 'tableCell',
+              attrs: {
+                colspan: 1,
+                rowspan: 1,
+                colwidth: null,
+              },
+              content: [
+                {
+                  type: 'paragraph',
+                  content: [
+                    {
+                      text: 'R3C2',
+                      type: 'text',
+                    },
+                  ],
+                },
+              ],
+            },
+            {
+              type: 'tableCell',
+              attrs: {
+                colspan: 1,
+                rowspan: 1,
+                colwidth: null,
+              },
+              content: [
+                {
+                  type: 'paragraph',
+                  content: [
+                    {
+                      text: 'R3C3',
+                      type: 'text',
+                    },
+                  ],
+                },
+              ],
+            },
+          ],
+        },
+      ],
+    },
+    {
+      type: 'paragraph',
+    },
+  ],
+}; */
```

Comment:
(non-blocking) is this comment something that we should keep?

Reply:
@edodusi it helps to debug the richtext node for table offline without connecting to the storyblok api. Is there any reason not to have it?

Reply:
@alvarosabu it's probably better to have this in the README or a dedicated documentation because it's hard to maintain, like since it's a comment if this gets obsolete we won't have signals that tell us.

Or this could be in a unit test, so that if it changes the test fails.

But yeah, non blocking comment, if you think it's fine let's move on and keep it